src/Pages/Artists/ArtistsDetails/index.tsx

```diff
@@ -21,6 +23,8 @@ export interface ArtistsDetailsProps {
 
 interface ArtistsDetailsState {
     readonly descriptionLoaded: boolean;
+    readonly index: number;
+    readonly routes: Array<any>;
```

Comment:
Czy używamy gdzieś `index` i `routes`?

Reply:
Mogę to usunąć w moich poprawkach? Bo z tego co widzę to nie jest nigdzie używane.

Reply:
Tak, ten komponent TabBar potrzebuje mieć to w state